2333/2/libs/analyzer.rb

```diff
@@ -0,0 +1,47 @@
+require_relative 'handler'
+require 'terminal-table'
+require_relative './analyzer_printer.rb'
+require_relative './names_searcher.rb'
+
+# VersusBattle text analyzer
+class Analyzer
+  def initialize(rappers, take_value = nil, selected_name = nil)
+    @rappers = rappers
+    @take_value = take_value.to_i
+    @name = selected_name
+  end
+
+  def top_bad_words
+    AnalyzerPrinter.print_top_bad_words(analyze_rappers)
+  end
+
+  def top_words
+    exit unless names_include_name?
+    texts = Handler.delete_prepositions(@rappers, @name)
```

Comment:
Hm... Instead of creating a variable like this, we may extract this to a separate method `#texts_wo_stopwords`.